src/nfv_test_api/v2/data/namespace.py

```diff
@@ -0,0 +1,45 @@
+"""
+       Copyright 2021 Inmanta
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+"""
+from typing import Optional
+
+from pydantic import BaseModel
+
+from .base_model import IpBaseModel
+from .common import SafeName
+
+
+class NamespaceCreate(BaseModel):
+    """
+    Input for creating a network namespace
+    """
+
+    name: SafeName  # type: ignore
+    ns_id: Optional[int]
```

Comment:
The `ns_id` is the namespace id, this is something that can be set by the user, if it is not provided, it will automatically be assigned.  I am not sure why exactly this is an option, but the `ip netns` command allows to set it so I let it go through the api.

Do you have a preference about the way it should be documented?  With `pydantic.Field()` or just a docstring on the class?
```python
class NamespaceCreate(BaseModel):
    """
    Input for creating a network namespace

    :ivar ns_id: The optional namespace id, if none is provided, one will be picked in the pool of available ids
    """

    name: SafeName  # type: ignore
    ns_id: Optional[int] = Field(
        description="The optional namespace id, if none is provided, one will be picked in the pool of available ids"
    )

```
would look like
![image](https://user-images.githubusercontent.com/20844316/137080641-7c021006-6894-4ec3-842b-4390aa9a2718.png)

Reply:
Most other parameters get docs in swagger, so I guess docstring is more than fine.